installPhase.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+if [ -v out ]; then
+  echo "Will install package to $out (defined outside installPhase.sh script)"
+else
+  out="./result"
+  echo "Will install package to $out"
+fi
```

Comment:
@kobyhallx What's the reasoning behind this? The flake will install to `./result` but if I just run `./build.sh` then I get the package in `./outputs/out` which is a bit annoying.

I also can't see where the `$out` env var is being set.

Reply:
Ok, why do we choose to have different behaviour here for nix and non-nix workflows?